src/literals.py

```diff
@@ -37,6 +37,7 @@
 
 S3_REL_NAME = "s3-credentials"
 S3_BACKUPS_PATH = "zookeeper_backups"
+S3_BACKUPS_LIMIT = 20
```

Comment:
Copy pasted from my comment to Marc: 
> This is the maximum number of backups to be displayed with the list-backups action. 20 is a complete arbitrary choice, there is no meaning behind it. I figured that users are more likely to care about the latest backups than the older ones.
The reason why we limit this is outlined in the method in the backup manager. As it is, we will make at most 21 queries to S3


We can increase this threshold, but this makes the action longer and longer to run. We need a different approach to metadata storing rather than having X yaml files on the object storage, but that's a discussion that would probably impact all DP products

Reply:
We could quickly add an action parameter "max-num" or something similar for this threshold, but that would already set the tone for other products

Reply:
Maybe worth raising at Office hours to check the temperature on this. I'm thinking that we could make the action paginable, like saying "start epoch" and then we go back on n items, or something like this

Reply:
anyhow, I reckon this could have been something where we want to be standard, and it is not part of the current PR (that should only implement standard UX)

Reply:
Added an item for next office, so I'll resolve this thread and merge this PR and the one on VM. Thank you for your thoughts